src/code/z_sram_NES.c

```diff
@@ -1424,23 +1423,23 @@ void func_801457CC(GameState* gameState, SramContext* sramCtx) {
                     Lib_MemCpy(&gSaveContext, sramCtx->saveBuf, gFlashSaveSizes[sp64]);
 
                     // test checksum of main save
-                    oldCheckSum = gSaveContext.save.saveInfo.checksum;
+                    phi_s2 = gSaveContext.save.saveInfo.checksum;
                     gSaveContext.save.saveInfo.checksum = 0;
                     newCheckSum = Sram_CalcChecksum(&gSaveContext, gFlashSaveSizes[sp64]);
-                    gSaveContext.save.saveInfo.checksum = oldCheckSum;
+                    gSaveContext.save.saveInfo.checksum = phi_s2;
 
-                    if (CHECK_NEWF(gSaveContext.save.saveInfo.playerData.newf) || (oldCheckSum != newCheckSum)) {
+                    if (CHECK_NEWF(gSaveContext.save.saveInfo.playerData.newf) || (newCheckSum != phi_s2)) {
                         // checksum didnt match, try backup save
                         sp6E = 1;
 
                         if ((gSaveContext.save.saveInfo.playerData.newf[0] == 'Z') &&
-                            (gSaveContext.save.saveInfo.playerData.newf[1] == 'E')) {
-                            phi_s2 = false;
-                        }
-
-                        // phi_s2 = false;
+                            (gSaveContext.save.saveInfo.playerData.newf[1] == 'E') &&
+                            (gSaveContext.save.saveInfo.playerData.newf[2] == 'L') &&
+                            (gSaveContext.save.saveInfo.playerData.newf[3] == 'D') &&
+                            (gSaveContext.save.saveInfo.playerData.newf[4] == 'A')) {}
```

Comment:
Of course. Why wouldn't it be like this? :upside_down_face:

Reply:
Memes aside, does the `CHECK_NEWF` macro works here?

Reply:
It does not, I tried.